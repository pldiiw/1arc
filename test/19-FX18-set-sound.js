'use strict';

let engine = require('engine').initialize();
const setSound = require('instructions').setSound;

engine.data[5] = 42;
engine.data[6] = 12;

const engine1 = setSound(engine, 5);
const engine2 = setSound(engine1, 6);

console.log(engine.sound);
console.log(engine1.sound);
console.log(engine2.sound);
