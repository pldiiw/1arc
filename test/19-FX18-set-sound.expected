0
42
12
